### ISSUE TITLE
Nil conversion on connection_hard_timeout (v 0.4.0)

### ISSUE THREAD

**Issue by freedomofkeima** [tagomoris/fluent-plugin-secure-forward#35]
The newest version caused fluentd to crash.

```
2016-03-30 14:54:00 +0000 [error]: unexpected error error_class=TypeError error=#<TypeError: can't convert nil into an exact number>
2016-03-30 14:54:00 +0000 [error]: /opt/td-agent/embedded/lib/ruby/gems/2.1.0/gems/fluent-plugin-secure-forward-0.4.0/lib/fluent/plugin/output_node.rb:333:in `+'
...
2016-03-30 14:54:00 +0000 [warn]: /opt/td-agent/embedded/lib/ruby/gems/2.1.0/gems/fluent-plugin-secure-forward-0.4.0/lib/fluent/plugin/output_node.rb:331:in `connect'
2016-03-30 14:54:00 +0000 [info]: process finished code=0
2016-03-30 14:54:00 +0000 [warn]: process died within 1 second. exit.
```

It seems the default value of `connection_hard_timeout` is set to `nil` [here](https://github.com/tagomoris/fluent-plugin-secure-forward/commit/f7de545515dbad8e8cdc9776cf950c4f396859e7#diff-9822b4b84875ff00780f71f0faa3c575R26) while this [line](https://github.com/tagomoris/fluent-plugin-secure-forward/commit/f7de545515dbad8e8cdc9776cf950c4f396859e7#diff-02c1517105a46b18172c993e13647c42R333) and this [line](https://github.com/tagomoris/fluent-plugin-secure-forward/commit/f7de545515dbad8e8cdc9776cf950c4f396859e7#diff-02c1517105a46b18172c993e13647c42R348) try to compare its value.

**Comment by tagomoris** [comment]
@freedomofkeima It's actually a bug. Thank you for report!
I'd fixed it and released v0.4.1. Could you try it?

**Comment by freedomofkeima** [comment]
Thanks, it's starting properly now :smile:
